### ISSUE TITLE
resolver: dirty free-form search blocks structured search

### ISSUE THREAD

**Issue by danielhochman** [lyft/clutch#144]
**Description**

If I fill in something for the free-form section and empty it out, the form does not recognize that it should use the structured search and instead enforces require on the free-form field.

**Expected Behavior**

1. Type something in the free-form search box.
1. Empty the free-form search box
1. Fill in the structured portion of the resolver
1. Submit

**Actual Behavior**

![Screenshot from 2020-07-29 14-21-39](https://user-images.githubusercontent.com/4712430/88843536-e5e52f00-d1a6-11ea-905a-2a1d4a4dd51e.png)

**Version**
`main`

**Comment by dschaller** [comment]
@danielhochman I can't reproduce this. Are you sure you didn't click into the free-form text field and then clicked continue?

**Comment by danielhochman** [comment]
hmm maybe that's what happens but i still don't see why it would choose to submit the upper form if focused but empty?

i may type something, submit, not found, fill in the bottom form, then empty the upper form and click submit.

maybe the real fix is to disable the form that is not in-use, that would make that scenario impossible.

**Comment by dschaller** [comment]
Yeah, this was a design choice when we decided to remove the toggle button and add both the free-form input field and schema fields onto the same page. The way we denote which to use is by the last selected field. The issue with disabling it is that the user won't have a way to switch back to that field if they want.

Maybe Kanchan will have some idea on how to improve this.

**Comment by danielhochman** [comment]
Was thinking the way to switch back to it would be to empty out the other form (could be a reset button). This is way clearer than letting input exist in both forms simultaneously IMO. Let's run it by Kanchan for sure and see what our options are, good case/design study.
